filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderFactorySpec.scala

```diff
@@ -0,0 +1,31 @@
+package cromwell.filesystems.blob
+
+import com.typesafe.config.ConfigFactory
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class BlobPathBuilderFactorySpec extends AnyFlatSpec with Matchers {
+
+  it should "parse configs for a functioning factory" in {
+    val endpoint = BlobPathBuilderSpec.buildEndpoint("coaexternalstorage")
```

Comment:
Let's not check in this storage account name. 

It might be worth taking another look at this test as part of our conversations about overall testing strategy, but I think it's fine to merge for now.